app/jobs/search/index_job.rb

```diff
@@ -0,0 +1,14 @@
+module Search
+  class IndexJob < ApplicationJob
```

Comment:
Can we get some specs written for this job?

Reply:
Yep, the pr is still WIP.